styles/dialog/fight.scss

```diff
@@ -7,9 +7,9 @@
     }
 
     .control-grid {
-        display: grid;
-        grid-template-columns: 1fr 1fr;
-        column-gap: 10px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
```

Comment:
I'm not sure why this change was necessary. The only thing that is visually different is the fact there is no `column-gap` anymore.

Reply:
Also it forms sort of a control-panel to which it's possible to add other buttons and the space will be divided evenly among them without having to every change css